src/views/AccountSettings.vue

```diff
@@ -51,8 +57,14 @@ export default {
 		Content,
 		EditorSettings,
 		Navigation,
+		MailboxPicker,
 		SignatureSettings,
 	},
+	data() {
+		return {
+			mailbox: '',
```

Comment:
```suggestion
			mailboxId: '',
```

because it's just the ID and not the object

Reply:
Ok, you are absolutely right...